src/styles/Product.module.scss

```
@@ -0,0 +1,36 @@
+@import "variables";
+* {
+    box-sizing: border-box;
+}
+.product {
+    @include square(500px, 100px);
+    cursor: pointer;
+    position: relative;
+    text-align: center;
+    margin: 30px auto;
+    font-family: "Open Sans";
+    &_title {
```

внимательно про БЭМ, тут 
```suggestion
    &__title {
```

---

ниже в этом файле также

---

не вижу, чтобы были вынесены цвета в переменные, только миксины, добавить обязательно!